src/components/entity/ha-state-label-badge.html

```diff
@@ -14,6 +14,19 @@
 
       ha-label-badge {
         --ha-label-badge-color: var(--label-badge-red, #DF4C1E);
+        --ha-label-badge-label: {
```

Comment:
Since it's only 1 property, let's use a CSS property instead of mixin. Properties are natively supported by the platform while mixins have to get polyfilled.